web/styles/rendered_markdown.css

```diff
@@ -227,7 +227,7 @@
         margin-left: 2px;
         margin-right: 2px;
         display: inline-block;
-        margin-bottom: 1px;
+        margin-top: 5px;
```

Comment:
We don't want to modify it for the whole app. Just the help overlay.

Reply:
If I do 
`p time{
        margin-top: 5px;
}` would that be okay ? As I cann't see anywhere else changes being made .

Reply:
You should definite it for time in `informational_overlays.css`.

Reply:
Since this not a bug outside fo it.
<img width="524" alt="Screenshot 2024-07-15 at 6 48 12 PM" src="https://github.com/user-attachments/assets/ee8cadac-c5a0-4b16-a480-30f836c75439">